client/src/components/modals/pick-entity-or-conversion.vue

```diff
@@ -113,6 +105,9 @@ export default class PickEntityOrConversion extends Vue {
   get location () {
     return newReqModule.location
   }
+  get locationText () {
+    return newReqModule.locationText === 'BC' ? 'British Columbia' : newReqModule.locationText
```

Comment:
Perfect.
Whenever practical, don't have logic in the template.

Reply:
Definitely!